tests/test_routing.py

```diff
@@ -864,24 +864,6 @@ def test_mount_asgi_app_with_middleware_url_path_for() -> None:
         mounted_app_with_middleware.url_path_for("route")
 
 
-def test_add_route_to_app_after_mount(
```

Comment:
I'd keep this test as long as the feature code exists.

Reply:
I'll readd it with the warning filter.